### PR TITLE
Project navigation update

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Updated Project navigation to use Ant Design.

## Related issue
N/A

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [X] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [X] Tests added (or description of how to test) for any new features.
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

File: src/main/webapp/resources/js/components/project/ProjectNav.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  return (
+    <PageHeader
+      title={window.project.label}
+      avatar={{ icon: <IconFolder /> }}
+      tags={[<RemoteProjectStatus key="remote" />].filter((f) => f !== null)}
+    >
+      <Content>
+        <Menu mode="horizontal" selectedKeys={[current]}>
+          <Item key="samples">
+            <a href={setBaseUrl(`projects/${window.project.id}/samples`)}>
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/components/project/ProjectNav.jsx:27]: Set `window.project.id` as a constant as it is used multiple times

File: src/main/webapp/resources/js/components/project/RemoteProjectStatus.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/components/project/RemoteProjectStatus.jsx:1]: File comments

Comment (joshsadam) [reply src/main/webapp/resources/js/components/project/RemoteProjectStatus.jsx:1]: Updated in 63bdc79ce23362f910cfd9f4b7843d4c10ff0743

File: src/main/webapp/resources/js/components/project/ProjectNav.jsx (hunk 1 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/components/project/ProjectNav.jsx]: Updated in 1fbb0cd49c9077eb611714fd1e0d1fbfa9525180

**Pull request merged**